tutorials/week1-installing-postman.md

```diff
@@ -0,0 +1,67 @@
+# Tutorial: How to Install and Use Postman
+
+## Introduction to Postman
+
+Postman is a popular API testing tool that allows developers to build, test, and modify APIs quickly and easily. It provides a interface to interact with APIs and simplifies API development. Postman supports making requests like GET, POST, PUT, DELETE, and more.
```

Comment:
After this paragraph, do we want to add that you can make these API requests using programming language like Python (`requests` library) or Node (`axios`)? IP1 includes the following:

> You can send requests to the server using a tool like Postman, or by writing scripts that use axios to send requests to localhost:8000, as you did in the Async activity.

I suggest this for 2 reasons:
1. If this is a broader guide about API requests + Postman, more experienced students can review the API requests topics and then switch to the language of their choice.
2. It emphasizes the user-convenience that Postman provides.

More of a nit-picky suggestion, but something to consider